src/listener/unix_listener.rs

```diff
@@ -7,63 +7,44 @@ use std::fmt::{self, Display, Formatter};
 
 use async_std::os::unix::net::{self, SocketAddr, UnixStream};
 use async_std::prelude::*;
-use async_std::{io, path::PathBuf, task};
+use async_std::{io, task};
+use async_std::path::PathBuf;
 
 /// This represents a tide [Listener](crate::listener::Listener) that
 /// wraps an [async_std::os::unix::net::UnixListener]. It is implemented as an
 /// enum in order to allow creation of a tide::listener::UnixListener
-/// from a [PathBuf] that has not yet been opened/bound OR from a bound
+/// from a [`PathBuf`] spec that has not yet been bound OR from a bound
```

Comment:
oops; I copy pasted the `TcpListener` impl and changed it. Looks like some of the wording came over. That shouldn't be too big a deal I hope.